src/components/Review/ChangeRequest.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { ChangeEvent } from 'react';
+
+import "./ChangeRequest.css";
+import TextField from '@material-ui/core/TextField';
+
+export interface ChangeRequestElement {
+    comment: string;
+    page: string;
+    line: string
+}
+
+export default function ChangeRequest({changeRequests, setChangeRequests, value, index}: {
+    changeRequests: Array<ChangeRequestElement>,
+    setChangeRequests: React.Dispatch<React.SetStateAction<ChangeRequestElement[]>>,
+    value: ChangeRequestElement,
+    index: number
+}) {
+    function handleChangeChangeRequestComment(event: ChangeEvent<HTMLTextAreaElement | HTMLInputElement>, index: number) {
+        const newArr = [...changeRequests];
+        newArr[index] = {...newArr[index], comment: event.target.value};
```

Comment:
I think you can just edit the object directly
```suggestion
        newArr[index].comment: event.target.value;
```